beets/ui/commands.py

```diff
@@ -1479,17 +1479,20 @@ def move_items(lib, dest, query, copy, album, pretend, confirm=False):
             show_path_changes([(obj.path, obj.destination(basedir=dest))
                                for obj in objs])
     else:
-        if confirm:
-            objs = ui.input_select_objects(
-                u'Really %s' % act, objs,
-                lambda o: show_path_changes(
-                    [(o.path, o.destination(basedir=dest))]))
+        if export:
+            util.copy(item.path, item.destination(basedir=dest))
```

Comment:
That's odd! Which test do the two runs disagree on?

To help debug the failures, sometimes it can be helpful to try deleting the code you added and confirming that all the tests pass, as expected. Then you can add changes in one at a time to see when things break.